Woof/Woof/ViewModel/SitterListViewModel.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+/// An interface is responsible for preparing and providing data for the `SitterListView`.
+final class SitterListViewModel: ObservableObject {
+    /// The sequence of sitters to display in view.
```

Comment:
nit:

technically, this is not a sequence in the given context :) 

```suggestion
    /// The list of sitters to show.
```

Reply:
Fixed.